src/openrct2/localisation/StringIds.h

```diff
@@ -1230,7 +1230,7 @@ enum : StringId
     STR_OBJECT_SELECTION_PATH_SIGNS = 3188,
     STR_OBJECT_SELECTION_FOOTPATHS = 3189,
     STR_OBJECT_SELECTION_PATH_EXTRAS = 3190,
-    STR_OBJECT_SELECTION_SCENERY_GROUPS = 3191,
+    STR_OBJECT_SELECTION_SCENERY_GROUPS = 2259,
```

Comment:
This is an odd re-use of an existing string id (`STR_RESEARCH_NEW_SCENERY_AND_THEMING`). I think I see why you did this, but it really should get a string of its own due to the different context. It may be fine in English, but other languages might be more context-sensitive. Please revert and edit string 3191 instead.